Warning about usage of execlp

Running make result in the following warning:
```
gcc -c -o obj/applicationLayer.o src/applicationLayer.c -I./include -lpthread
src/applicationLayer.c: In function ‘op_and_save’:
src/applicationLayer.c:287:17: warning: missing sentinel in function call [-Wformat=]
  287 |                 execlp(systemCall, systemCall, 0);
      |                 ^~~~~~
src/applicationLayer.c:290:9: warning: missing sentinel in function call [-Wformat=]
  290 |         execlp("echo", "echo", "command not found", 0);
      |         ^~~~~~
```

Can be fixed by replacing 0 with NULL or casting 0 to char *

--- a/src/applicationLayer.c
+++ b/src/applicationLayer.c
@@ -283,10 +283,10 @@ int op_and_save(char *systemCall, char *key) {
 
         for (int i = 0; i < sysCallsLength; i++) {
             if (strcmp(systemCall, sysCalls[i]) == 0) {
-                execlp(systemCall, systemCall, 0);
+                execlp(systemCall, systemCall, NULL);
             }
         }
-        execlp("echo", "echo", "command not found", 0);
+        execlp("echo", "echo", "command not found", NULL);
         return 1;
     } else {
         wait(0);

